demo/admin/src/common/ContentScopeProvider.tsx

```diff
@@ -32,8 +32,16 @@ export const ContentScopeProvider = ({ children }: Pick<ContentScopeProviderProp
         language: { value: contentScope.language, label: contentScope.language.toUpperCase() },
     }));
 
+    if (!user.allowedContentScopes[0]) {
```

Comment:
This is easier to comprehend:
```suggestion
    if (user.allowedContentScopes.length === 0) {
```

Reply:
changed here: 0030566fa184c31b8981db3cdb788497afe1c4b6